doc/classes/ProjectSettings.xml

```diff
@@ -2699,17 +2699,19 @@
 			[/codeblock]
 		</member>
 		<member name="threading/worker_pool/low_priority_thread_ratio" type="float" setter="" getter="" default="0.3">
+			If [member threading/worker_pool/use_system_threads_for_low_priority_tasks] is [code]false[/code], this value determines the ratio of [WorkerThreadPool]'s threads that will be reserved for low-priority tasks. For example, if 10 threads are available and this value is set to [code]0.3[/code], 3 of the worker threads will be reserved for low-priority tasks. The actual value won't exceed the number of CPU cores minus one, and if possible, at least one worker thread will be dedicated to low-priority tasks.
```

Comment:
This looks good. When you rebase, you'll have to remove the reference to `use_system_threads_for_low_priority_tasks`, which is now gone (and, so to say, hardcoded to `false`).

Reply:
Forgot to mention that this is doned now